### PR TITLE
factory-package-news: fixes for Tumbleweed

### PR DESCRIPTION
**Pull request by ggardet**
This will fixes https://github.com/openSUSE/openSUSE-release-tools/issues/2216

### CODE REVIEW

**Comment by coolo** [comment]
I can merge, but I can't say if this is correct. @lnussel perhaps

File: factory-package-news/factory-package-news.py [hunk 1]
```diff
@@ -102,7 +102,9 @@ def _getdata(h):
                 if not iso.is_open() or fd is None:
                     raise Exception("Could not open %s as an ISO-9660 image." %  arg)
 
-                for path in ['/suse/x86_64', '/suse/noarch', '/suse/aarch64', '/suse/s390x']:
+                # On Tumbleweed, there is no '/suse' prefix
+                for path in ['/suse/x86_64', '/suse/noarch', '/suse/aarch64', '/suse/s390x',
+                             '/x86_64',      '/noarch',      '/aarch64',      '/s390' ]:
```
Comment (lnussel) [inline factory-package-news/factory-package-news.py]: s390x I suppose

**Comment by lnussel** [comment]
it is correct, actually someone hot fixed it in a similar way on openqa.o.o years ago

File: factory-package-news/factory-package-news.py (hunk 1 above)
Comment (ggardet) [reply factory-package-news/factory-package-news.py]: Good catch!

**Comment by ggardet** [comment]
@lnussel are there other hot fixes to merge back?

**Comment by lnussel** [comment]
no, just that

**Pull request merged**